CharacterCreation/04_Backgrounds.txt

```diff
@@ -0,0 +1,247 @@
+| Underworld Contacts  | Criminal  | Gain advantage when bartering with criminals                                  |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| You innately understand how underworld dealings work, and won't easily be swindled out of a good deal.           |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| Caveat Emptor        | Trade     | Gain 6 skill points in Insight.                                               |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| You've bought enough lemons to know when someone's pitching a bad deal. Spotting over-selling is easy.           |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| Well-Traveled        | Trade     | Gain 6 skill in Navigation and pick an additional dialect that you know.      |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| Oh ya, I had a collegue from these parts. Made a killing last time I was here selling "yoshgas." Don't worry     |
+| about it, it doesn't translate all that well.                                                                    |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| Hunter's Bag of Tricks | BountyHunter | Gain license, L2 chest armor plate, and 5 HD zip cuffs.                     |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| You have a legal license for bringing in bounties, dead or alive. You've got some other tools too.               |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| Friends in low places | BountyHunter | You gain +6 skill in Knowledge - Underworld.                              |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| You've brought in enough criminals to have an idea how the seedier side of the tracks opperates.                 |
++----------------------+-----------+-------------------------------------------------------------------------------+
+
+==== Clothing and Supplies ====
+
+Casual Clothing:
+-A t-shirt and jeans
+-A shirt, pants and a leather jacket
+-A trenchcoat, shirt and capris
+-Athletic-wear (such as sport-specific clothes)
+-A sundress
+-A hoddie and sweats or jeans
+-Cold-weather clothing
+-Waterproof jacket, pants and boots
+-Loose, breatheable top and pants with a wide-brimmed hat
+-Shorts and a tank top
+
+Professional Clothing
+-Khakis and a polo shirt or button-down and sweater
```

Comment:
"-Khakis, a polo shirt or button-down, and sweater"
Gotta use that sweet, sweet oxford comma here